### PR TITLE
feat: Optimisation of code

### PR DESCRIPTION
**Pull request by IbrahimBagalwa**
# PR Description

Change the previous image with the default one when we move the mouse

# Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you  Make sure you check every completed task. list them in this style

- [x] Change the Previous img with the default
- [x] put the mouve in the center of the img

# Description of tasks that were expected to be completed and that are not yet complited

- [x] None

# How this maybe tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependences 
2. run `npm run dev`

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: components/sections/Gallery.jsx [hunk 1]
```diff
@@ -15,6 +16,7 @@ export default function Gallery() {
           );
         })}
       </div>
+      <img src="/assets/imgs/default.png" alt="" className="def hidden" />
```
Comment (Yvad60) [inline components/sections/Gallery.jsx]: Can we use the Image component from Next here

Comment (Yvad60) [inline components/sections/Gallery.jsx]: Also in the this file where we are mapping the `gallery` array can we change the name of the element in the call-back to something more descriptive than `im` I think `image` or something similar would be better

File: utils/moveImage.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
 const TweenMax = require("/public/cdnjs/TweenMax.min.js");
+
```
Comment (Yvad60) [inline utils/moveImage.js:2]: this is Not related to this file I Just wanted to leave the comment here.

Can we remove the `GallerySimulation.jsx` if we no longer need it

**Review by Yvad60 — CHANGES_REQUESTED** [review]
Good one guys I noticed a lot work and done to the Gallery section. 
Below I suggested some changes we can include

File: components/sections/Gallery.jsx (hunk 1 above)
Comment (IbrahimBagalwa) [reply components/sections/Gallery.jsx]: In the canvas tag when you draw with the Image component you will get an empty design.

File: utils/moveImage.js [hunk 3]
```diff
@@ -20,9 +27,8 @@ function movingImg() {
   let gravityX, gravityY;
   let randImages = document.querySelectorAll("#randsImages .img");
   let randIndex = 0;
-
+  let def = document.querySelector(".def");
```
Comment (serapieTuyishime) [inline utils/moveImage.js]: def is not a very descriptive name

File: utils/moveImage.js [hunk 4]
```diff
@@ -1,7 +1,14 @@
 const TweenMax = require("/public/cdnjs/TweenMax.min.js");
+
 function movingImg() {
+  let prev = {
```
Comment (serapieTuyishime) [inline utils/moveImage.js]: prev is not a descriptive name, maybe uses something as `previous image`

**Review by serapieTuyishime — CHANGES_REQUESTED** [review]

File: components/sections/Gallery.jsx (hunk 1 above)
Comment (chrisostome01) [reply components/sections/Gallery.jsx]: I guess if it was even possible it would have some lazy loading during the image appearance

**Review by Yvad60 — APPROVED** [review]
I think it better now 
approved !

**Review by serapieTuyishime — APPROVED** [review]
Impressive.

**Review by chrisostome01 — APPROVED** [review]
